finetune/encoding.py

```diff
@@ -156,7 +156,7 @@ def _encode(self, texts, labels=None, verbose=True):
                 ])
                 token_start = raw_text.find(token.text, token_start)
                 assert len("".join(bpe_toks).replace("</w>", "")) == len(token.text.strip())
-                subtoken_positions = np.cumsum([len(tok.replace("</w>", '')) for tok in bpe_toks]) + token_start
+                subtoken_positions = np.cumsum([len(tok.replace("</w>", ' ')) for tok in bpe_toks]) + token_start
```

Comment:
sneaky off by one error

Reply:
Worth updating the assertion to reflect this. eg.
```
assert len("".join(bpe_toks).replace("</w>", " ")) == len(token.text) 
```